pkg/semver/semver.go

```diff
@@ -67,3 +67,23 @@ func (v *Version) SamePrerelease(v2 *Version) bool {
 func (v *Version) Equal(v2 *Version) bool {
 	return v.SamePrerelease(v2) && v.Buildmetadata == v2.Buildmetadata
 }
+
+func (v *Version) GreaterThan(v2 *Version) bool {
+	if v2 == nil {
+		return true
+	}
+	if v.Major > v2.Major {
+		return true
+	}
+	if v.Minor > v2.Minor {
```

Comment:
I don't think this is right
What about v is `v1.2.0` and v2 is `v2.1.0`
I would recommend writing tests for these methods